consensus/ibft/consensus_backend.go

```diff
@@ -3,18 +3,18 @@ package ibft
 import (
 	"context"
 	"fmt"
-	"math"
 	"time"
 
 	"github.com/0xPolygon/go-ibft/messages"
+	"github.com/0xPolygon/go-ibft/messages/proto"
 	"github.com/0xPolygon/polygon-edge/consensus"
 	"github.com/0xPolygon/polygon-edge/consensus/ibft/signer"
 	"github.com/0xPolygon/polygon-edge/helper/hex"
 	"github.com/0xPolygon/polygon-edge/state"
 	"github.com/0xPolygon/polygon-edge/types"
 )
 
-func (i *backendIBFT) BuildProposal(blockNumber uint64) []byte {
+func (i *backendIBFT) BuildEthereumBlock(blockNumber uint64) []byte {
```

Comment:
I remember that we called it `EthereumBlock` but what's the motivation for this name? why not call it simply `Block`?

Reply:
Renamed to BuildProposal at f3cd3dc5c2a57f3820b5362e50b650689a512c43